common/had_map_utils/CMakeLists.txt

```diff
@@ -5,27 +5,31 @@ cmake_minimum_required(VERSION 3.5)
 ### Export headers
 project(had_map_utils)
 
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Werror)
+endif()
+
 find_package(ament_cmake_auto REQUIRED)
 find_package(CGAL REQUIRED COMPONENTS Core)
-#find_package(Eigen3 REQUIRED)
+find_package(Eigen3 REQUIRED)
 ament_auto_find_build_dependencies()
 
-# Disable warnings due to external dependencies (Eigen)
-#include_directories(SYSTEM ${EIGEN3_INCLUDE_DIR})
-
 ament_auto_add_library(${PROJECT_NAME} SHARED
-  include/had_map_utils/had_map_utils.hpp
-  include/had_map_utils/had_map_computation.hpp
-  include/had_map_utils/had_map_conversion.hpp
-  include/had_map_utils/had_map_query.hpp
-  include/had_map_utils/had_map_visualization.hpp
-  include/had_map_utils/visibility_control.hpp
   src/had_map_utils.cpp
   src/had_map_computation.cpp
   src/had_map_conversion.cpp
   src/had_map_query.cpp
   src/had_map_visualization.cpp)
 
+target_include_directories(${PROJECT_NAME} SYSTEM PUBLIC
```

Comment:
Don't you need to keep this comment? `# Disable warnings due to external dependencies`

Reply:
fixed at https://github.com/autowarefoundation/autoware.universe/pull/198/commits/7be9c892ce497f820b56917ed471747f81aa5d7b.